internal/services/databricks/parse/databricks_workspaces_test.go

```diff
@@ -0,0 +1,229 @@
+package parse
```

Comment:
this file shouldn't be needed anymore, since it's available within `go-azure-sdk`?

Reply:
Fixed.